tests/network-sriov

```diff
@@ -101,21 +101,21 @@ lxc profile device add default eth0 nic nictype=sriov parent="${parentNIC}" name
 # Launch a few VMs.
 # Do this first before containers to ensure VF free search handles VFs unbound from host.
 echo "==> VM on default VLAN"
-lxc init ubuntu-daily:23.10 v1 --vm
+lxc init ubuntu-daily:24.04 v1 --vm
```

Comment:
@simondeziel a thought for another time, but it'd be nice to be able to change this for all tests with an env var, and then default to a specific version in a single place in the code.

Reply:
I've never seen the need for this as it's easy to change (`sed 's/:22\.04/:24.04/'`) either globally or on a single `tests/` file, IMHO. However if that is of use to you, sure. You'd like just the image name to be changeable? Would `$testImg` be a good variable name?

Reply:
I suppose something like `export TEST_IMG=ubuntu-daily:24.04` would be good.